contracts/contracts/coordination/Coordinator.sol

```diff
@@ -76,7 +74,7 @@ contract Coordinator is Initializable, AccessControlDefaultAdminRulesUpgradeable
         uint16 threshold;
         bool aggregationMismatch;
         //
-        IEncryptionAuthorizer accessController;
+        address stub1; // former accessController
```

Comment:
this remains so that the contract can be upgraded right?

Reply:
Being upgradeable means that current storage has this layout, so any change needs to be aware of this previous layout and avoid it. @vzotova usually names this something like `stub`, also used for slots reserved for future use.